src/nunit.analyzers.tests/ClassicModelAssertUsage/ClassicModelAssertUsageAnalyzerTests.cs

```diff
@@ -53,148 +51,173 @@ public void VerifySupportedDiagnostics()
         }
 
         [Test]
-        public async Task AnalyzeWhenThatIsUsed()
+        public void AnalyzeWhenThatIsUsed()
         {
-            await TestHelpers.RunAnalysisAsync<ClassicModelAssertUsageAnalyzer>(
-                $"{ClassicModelAssertUsageAnalyzerTests.BasePath}{(nameof(this.AnalyzeWhenThatIsUsed))}.cs",
-                Array.Empty<string>());
+            var testCode = TestUtility.WrapClassInNamespaceAndAddUsing(@"
+    public sealed class AnalyzeWhenThatIsUsed
+    {
+        public void Test()
+        {
+            Assert.That(true, Is.True);
+        }
+    }");
+            AnalyzerAssert.Valid<ClassicModelAssertUsageAnalyzer>(testCode);
```

Comment:
Nitpick: I usually use the overload `AnalyzerAssert.Valid(analyzer, testCode);` here. Where is a field. Reason I prefer this overload is that the tests are more copy paste friendly that way. 😃

Reply:
Copy paste friendly as you only need to update the field in one place. No biggie keeping it as-is is fine of course.

Reply:
Got it :+1: